src/render/tutorials/chrome-plugin/2_2.html.md

```diff
@@ -0,0 +1,495 @@
+
+```hocon
+fetch-offers-cb {
+  factory = cb
+  doAction = fetch-offers
+
+  config.circuitBreakerOptions {
+    maxFailures = 3
+    timeout = 500
+    resetTimeout = 20000
+  }
+}
+```
+
+The circuit breaker calls the `fetch-offers` action inside of itself (`doAction` property). If the action failed then the circuit breaker assumes something is wrong and won't actually call the action next time it's invoked. It'll just assume failure instead.
+
+We defined the following properties:
+- `maxFailures` - number of failures it takes for circuit breaker to assume the action is not working
+- `timeout` - the circuit breaker will assume failure after this time
+- `resetTimeout` - how long the circuit breaker should wait until it'll allow the action to execute again
+
+Now we need to adjust the task. This is how it should look like now:
+
+```
+payments-task {
+  action = fetch-user-info
+
+  on._success {
+    actions = [
+      {
+        action = fetch-payment-providers
+      },
+      {
+        action = fetch-offers-cb
+      }
+    ]
+
+    on {
+      _success {
+        action = te-pebble
+      }
+      _error {
+        action = te-pebble
+      }
+    }
+  }
+}
+```
+
+Firstly, we call `fetch-offers-cb` instead of `fetch-offers` in the composite. That's because the `fetch-offers` action will be called inside of `fetch-offers-cb`. Also, since we expect the task to fail, the composite should handle the `_error` transition. It will be called if any subtask ends with anythink different then success. In this case we still want to render the page so we call the `te-pebble` action. 
+
+When we reload the page we can see that the Offers section is empty. However the Payment Providers section should still be filled with data. The tool will show this graph:
+
+![Missing Node](/img/blog/chrome-extension/missing-node.png)
+
+There's a lot of going on here. The `fetch-offers-cb` node is blue. That's because it returned a `_fallback` transition. It's a custom transition. Transitions are nothing more than strings. `_success` and `_error` are standard and they display respectively as green and red. Anything else is a custom transition and is displayed as blue. If we click on the `fetch-offers-cb` node we can see why it returned fallback: `Exception: operation timeout`.
+
+![Circuit Breaker Details](/img/blog/chrome-extension/cb-details.png)
+
+The yellow node with an exclamation mark is a so called "missing node". It doesn't actually exist during processing. It's just displayed in the tool as a way to catch the developer's attention. It means that a transition was triggered (in this case `_fallback`) but there was no node to handle it. Because of that, the `_fallback` state was returned to the composite and thus the composite ended with an error (composites can only end with either `_success` or `_error`).

```

Comment:
```suggestion
The yellow node with an exclamation mark is a so-called "missing node". It doesn't exist during processing. It's just displayed in the tool as a way to catch the developer's attention. It means that a transition was triggered (in this case `_fallback`) but there was no node to handle it. Because of that, the `_fallback` state was returned to the composite and thus the composite ended with an error (composites can only end with `_success` or `_error`).
```